source/connect/csot.txt

```diff
@@ -15,32 +15,37 @@ Limit Server Execution Time
    :values: reference
  
 .. meta::
-   :keywords: error, blocking, thread, task
+   :keywords: error, blocking, thread, task, code examples
 
 Overview
 --------
 
-When you use the {+driver-short+} to perform a server operation, you can also
+In this guide, you can learn how to use the ``timeout_ms`` option to set a 
+timeout for server operations. 
+
+When you use the {+driver-short+} to perform a server operation, you can
 limit the duration allowed for the server to finish the operation. To do so,
 specify a **client-side operation timeout (CSOT)**. The timeout applies to all
 steps needed to complete the operation, including server selection, connection
-checkout, and server-side execution. When the timeout expires, the
-{+driver-short+} raises a timeout exception.
+checkout, and server-side execution. If the timeout expires before the operation 
+completes, the {+driver-short+} raises a timeout exception.
 
-timeout_ms Option
------------------
+``timeout_ms`` Option
+---------------------
```

Comment:
S: remove monospacing from titles per [SG](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#:~:text=Don%27t%20apply%20font%20treatments%20(bold%2C%20italics%2C%20or%20monospace)%20to%20text%20in%20a%20title%20or%20heading.)